cmd/rukpakctl/cmd/run.go

```diff
@@ -73,7 +73,6 @@ one version to the next.
 			}
 			_, err := r.Run(ctx, bundleDeploymentName, os.DirFS(bundleDir), rukpakctl.RunOptions{
 				BundleDeploymentProvisionerClassName: bundleDeploymentProvisionerClassName,
-				BundleProvisionerClassName:           bundleProvisionerClassName,
```

Comment:
Remove the `bundleProvisionerClassName` variable and flag configuration for this field as well?